java/client/src/main/java/glide/api/commands/BitmapBaseCommands.java

```diff
@@ -515,6 +515,46 @@ CompletableFuture<Long> bitop(
      */
     CompletableFuture<Long[]> bitfield(String key, BitFieldSubCommands[] subCommands);
 
+    /**
+     * Reads or modifies the array of bits representing the string that is held at <code>key</code>
+     * based on the specified <code>subCommands</code>.
+     *
+     * @see <a href="https://valkey.io/commands/bitfield/">valkey.io</a> for details.
+     * @param key The key of the string.
+     * @param subCommands The subCommands to be performed on the binary value of the string at <code>
+     *     key</code>, which could be any of the following:
+     *     <ul>
+     *       <li>{@link BitFieldGet}.
+     *       <li>{@link BitFieldSet}.
+     *       <li>{@link BitFieldIncrby}.
+     *       <li>{@link BitFieldOverflow}.
+     *     </ul>
+     *
+     * @return An <code>array</code> of results from the executed subcommands.
+     *     <ul>
+     *       <li>{@link BitFieldGet} returns the value in {@link Offset} or {@link OffsetMultiplier}.
+     *       <li>{@link BitFieldSet} returns the old value in {@link Offset} or {@link
+     *           OffsetMultiplier}.
+     *       <li>{@link BitFieldIncrby} returns the new value in {@link Offset} or {@link
+     *           OffsetMultiplier}.
+     *       <li>{@link BitFieldOverflow} determines the behaviour of <code>SET</code> and <code>
+     *           INCRBY</code> when an overflow occurs. <code>OVERFLOW</code> does not return a value
+     *           and does not contribute a value to the array response.
+     *     </ul>
+     *
+     * @example
+     *     <pre>{@code
+     * client.set(gs("sampleKey"), gs("A")); // gs("A") has binary value 01000001
```

Comment:
```suggestion
     * client.set(gs("sampleKey"), gs("A")).get(); // string "A" has binary value 01000001
```
This is a bit more clearer